### PR TITLE
Add supported URLs for Custom repositories

### PR DESCRIPTION
**Pull request by mjivraja**
The list of the supported URLs for adding custom repositories
was needed to make users aware. In addition to this, there is a
condition to using file:// protocol where the file permissions and SELinux
are setup to only allow file:// syncs within the
/var/lib/pulp/sync_imports/ directory.

What are the supported URL (http, https,file,nfs,git etc)
for custom repositories?

https://bugzilla.redhat.com/show_bug.cgi?id=2078031

* [X] I am familiar with the [contributing](https://github.com/theforeman/foreman-documentation/blob/master/CONTRIBUTING.md) guidelines.

Please cherry-pick my commits into:

* [ ] Foreman 3.4/Katello 4.6
* [ ] Foreman 3.3/Katello 4.5
* [ ] Foreman 3.2/Katello 4.4
* [ ] Foreman 3.1/Katello 4.3
* For Foreman 3.0 or older, please create a separate PR.
* We do not accept PRs for Foreman 2.3 or older.

### CODE REVIEW

**Comment by mjivraja** [comment]
Hello @maximiliankolb @evgeni 
Can you please have a look at this PR at your convenience...

File: guides/doc-Content_Management_Guide/topics/Importing_Content.adoc
Comment (maximiliankolb) [inline guides/doc-Content_Management_Guide/topics/Importing_Content.adoc:111]: I don't have a strong opinion about things in 2.5.

Some suggestions: we generally use in-line code instead of bold text to things like URLs (and protcols) and paths; also one sentence per line; and maybe reword to "{Project} supports three ...".

totally your call to leave as is or make adjustments.

Comment (mjivraja) [reply guides/doc-Content_Management_Guide/topics/Importing_Content.adoc]: Addressed all your suggestions as it falls into standard practice. :+1:

File: guides/doc-Content_Management_Guide/topics/Importing_Content.adoc
Comment (maximiliankolb) [inline guides/doc-Content_Management_Guide/topics/Importing_Content.adoc:113]:
```suggestion
If you are using a `file://` repositories, you have to place it under `/var/lib/pulp/sync_imports/` directory.
```

Comment (mjivraja) [reply guides/doc-Content_Management_Guide/topics/Importing_Content.adoc]: Addressed :+1:

File: guides/doc-Content_Management_Guide/topics/Importing_Content.adoc
Comment (evgeni) [inline guides/doc-Content_Management_Guide/topics/Importing_Content.adoc:113]:
```suggestion
If you are using a `file://` repository, you have to place it under `/var/lib/pulp/sync_imports/` directory.
```

**Review by evgeni — CHANGES_REQUESTED** [review]
one typo, otherwise LGTM!

are similar changes also required for the newer versions? I don't see such a comment there and would *think* that the same limitations apply?

**Comment by mjivraja** [comment]
> 

I guess it should apply to later versions. But, in the BZ it's not mentioned by a reporter. If it's applicable to 3.x versions, I can open the separate PR. :+1:

**Comment by mjivraja** [comment]
> one typo, otherwise LGTM!
> 
> are similar changes also required for the newer versions? I don't see such a comment there and would _think_ that the same limitations apply?

I guess it should apply to later versions. But, in the BZ it's not mentioned by a reporter. If it's applicable to 3.x versions, I can open the separate PR. +1

**Review by evgeni — APPROVED** [review]

**Comment by evgeni** [comment]
Huh, should have squashed the commits, ups.

And yes, please have a look at the newer docs, I think the structure is different, but the same comment should be added there.

**Comment by mjivraja** [comment]
> Huh, should have squashed the commits, ups.
> 
> And yes, please have a look at the newer docs, I think the structure is different, but the same comment should be added there.

I would have squashed it before the merge. I will be careful next time :grinning: 

Wil definitely look into the 3.x versions and file a separate PR. Thank you for this detail @evgeni

**Pull request merged**
